mpi-proxy-split/p2p_log_replay.cpp

```diff
@@ -77,16 +77,16 @@ updateCkptDirByRank()
   o << baseDir << "/ckpt_rank_" << g_world_rank;
   dmtcp_set_ckpt_dir(o.str().c_str());
 
-  if (!g_list || g_numMmaps == 0) return;
+  if (!g_list || !g_numMmaps) return;
```

Comment:
Is this part correct? Is it possible that g_numMmaps is not null, but points to an int* which does contain 0 segments? 

I guess it might be just checking if this is initialized. If so, one could choose a sentinel value of some kind.